ajax/libs/simpleGallery.js/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "simpleGallery.js",
```

Comment:
@dada0423 Why not use the same library name as upstream? https://github.com/fccoelho7/simpleGallery.js/blob/master/package.json#L2